src/schema/core.yaml

```diff
@@ -215,6 +220,8 @@ classes:
         range: integer
       num_23s:
         range: integer
+      num_tRNA:
```

Comment:
num_tRNA is not a valid camel_case slot name and should appear as `num_t_rna` instead